files/en-us/web/javascript/guide/control_flow_and_error_handling/index.html

```diff
@@ -55,7 +55,7 @@ <h3 id="Example">Example</h3>
 <p>Here, <code>{ x++; }</code> is the block statement.</p>
 
 <div class="notecard note">
-  <p><strong>Note:</strong> JavaScript before ECMAScript2015 (6<sup>th</sup> edition)
+  <p><strong>Note:</strong> JavaScript before ECMAScript2015 (sixth edition)
```

Comment:
```suggestion
  <p><strong>Note:</strong> JavaScript before ECMAScript2015 (6th edition)
```
(for consistency with other changes)